src/rust/madara/template/src/helper.rs

```diff
@@ -195,3 +195,13 @@ pub fn get_int_manga_id(manga_id: String, base_url: String, path: String) -> Str
 	let id = &id_html[id_html.find("manga_id").unwrap() + 11..id_html.find("\"};").unwrap()];
 	String::from(id)
 }
+
+pub fn get_lang_code() -> String {
```

Comment:
I think it'd be better to use an `Option<String>` here, so that way you don't have to use a mutable and can return None instead of another allocated string. Then, in the leviatan source, you can fall back to "en" as the default language.